src/control/cmd/daos_server/support.go

```diff
@@ -53,6 +54,13 @@ func (cmd *collectLogCmd) Execute(_ []string) error {
 		}
 	}
 
+	if err := cmd.loadConfig(); err != nil {
+		return errors.Wrapf(err, "failed to load config from %s",
+			cmd.configPath())
+	} else if cmd.configPath() != "" {
+		cmd.Infof("DAOS Server config loaded from %s", cmd.configPath())
+	}
```

Comment:
I don't understand why this is needed. The `collectLogCmd` struct embeds the `cfgCmd` struct which implements the `cfgLoader` interface, so this should happen automatically over in [main.go](https://github.com/daos-stack/daos/blob/master/src/control/cmd/daos_server/main.go#L167).

Reply:
That's what I had thought at as well. But this is the general problem I've had with this change from Day 1.
From what I can tell there is no global variable with this information. It's my understanding of why https://github.com/daos-stack/daos/blob/5ea9557d84e2a62c8bbf1fcdea62bbfc5b5dca5d/src/control/lib/support/log.go#L277 code exists.

Reply:
The entrypoint is here https://github.com/daos-stack/daos/blob/5ea9557d84e2a62c8bbf1fcdea62bbfc5b5dca5d/src/control/lib/support/log.go#L954
The call comes from another machine which doesn't have access to the config file.

Reply:
OK, I've checked this PR out to look at it locally. I still don't think this particular change is needed. I understand why you're loading the server config way down in `rsyncLog()`, but loading it here seems redundant. Have you verified that `cmd.configPath()`, which is used to set `params.Config` down around line 88, is actually wrong unless you load this here?

My concern with just letting this slide is that it sets a precedent. It shouldn't be necessary due to the config loading logic in main.go. If that logic is not working as expected, it should be fixed instead of working around it over here. If you have to work around it now, then the next developer who wants to add some stuff will see your changes and add another workaround, and eventually we've got a big(ger) pile of spaghetti code.

Reply:
That's correct. The params.Config comes the client of this RPC call.
I need the server side to inject it's settings in here somehow.

Reply:
The same function is getting called from dmg command too and we do not pass the server config to dmg command. 
https://github.com/daos-stack/daos/blob/12c4905b06ec13ab53375dbaaacce2031e772c8e/src/control/cmd/dmg/support.go#L162

Reply:
@kjacque @mjmac 
Ok, I think I see how this can/should be done.
The code in https://github.com/daos-stack/daos/blob/master/src/control/cmd/daos_server/config.go needs to move from the main package into a library. Then this code can access the config file loaded in main.go at server startup.
This is a lot bigger change than the one that's already in this PR. I would like to call that refactor out of scope for this change. (pretty please)

Reply:
Ok, finally understood what was asked. Thanks @mjmac
Done.